### PR TITLE
Add Solarkraft blogpost #1

### PR DESCRIPTION
**Pull request by thpani**

### CODE REVIEW

File: _posts/2024-06-14-solarkraft-part1.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  - "apalache"
+  - "formal-methods"
+  - "runtime-monitor"
+  - "smart-contracts"
+  - "solarkraft"
+  - "soroban"
+  - "specification"
+  - "stellar"
+  - "tla"
+  - "tlaplus"
```
Comment (thpani) [inline _posts/2024-06-14-solarkraft-part1.md:17]: Please check these

File: _posts/2024-06-14-solarkraft-part1.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+In the following posts of this series, we'll delve deeper into the world of runtime monitoring in Solarkraft with TLA+. We'll explore how to specify contract behavior using this language, dive into Solarkraft's unique specification and monitoring strategies, and see how runtime verification can be seamlessly integrated into the development workflow to build more secure and reliable smart contracts.
+
+_Development of Solarkraft was supported by the [Stellar Development Foundation][] with a generous Activation Award from the [Stellar Community Fund][] of 50,000 USD in XLM._
+
+
+[Apalache]: https://konnov.phd/portfolio/apalache/
+[Solarkraft]: https://thpani.net/solarkraft/
+[Audits]: https://thpani.net/audits/
+[Contact]: https://thpani.net/#contact
```
Comment (thpani) [inline _posts/2024-06-14-solarkraft-part1.md:123]: idk if you want these to link to your site or mine. feel free to change :)

File: _posts/2024-06-14-solarkraft-part1.md (hunk 1 above)
Comment (konnov) [reply _posts/2024-06-14-solarkraft-part1.md:17]: Looks good!

File: _posts/2024-06-14-solarkraft-part1.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  - "stellar"
+  - "tla"
+  - "tlaplus"
+---
+
+![]({{ site.baseurl }}/img/solarkraft.png)
+
+» **This guest post by [Thomas Pani][] first appeared [on his blog][part1].**
+
+_In this series of blog posts, I'm introducing [Solarkraft][], a TLA+-based runtime monitoring solution for [Soroban smart contracts][Soroban]. We will start easy, with an overview of smart contracts, their principal vulnerabilities, and the traditional model of securing smart contracts, and an overview of how Solarkraft proposes a new solution._
```
Comment (konnov) [inline _posts/2024-06-14-solarkraft-part1.md:24]:
```suggestion
_In this series of blog posts, we introduce [Solarkraft][], a TLA+-based runtime monitoring solution for [Soroban smart contracts][Soroban]. We will start easy, with an overview of smart contracts, their principal vulnerabilities, and the traditional model of securing smart contracts, and an overview of how Solarkraft proposes a new solution._  
```

File: _posts/2024-06-14-solarkraft-part1.md (hunk 2 above)
Comment (konnov) [reply _posts/2024-06-14-solarkraft-part1.md:121]: I think it's fine, if the inbounds go to you this time :)

**Review by konnov — APPROVED** [review]
Looks good, just a few tiny fixes :)

**Pull request merged**